packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -31,147 +32,48 @@ However, if an error occurs during sign-in, Okta may need to display an error pa
-
-### <span v-pre>`{{orgName}}`</span>
-
-Inserts the org name title.
-
-Example:
-
-```html
-<title>{{orgName}} - {{errorSummary}}</title>
-```
-
-### <span v-pre>`{{errorSummary}}`</span>
-
-Inserts the error title text.
+Use the code editor to modify any HTML, CSS, or JavaScript on the error page. See [Customization examples](#customization-examples) for snippets that you can update and use.
 
-Example:
-
-```html
-<h2 class="o-form-title">{{errorSummary}}</h2>
-```
-
-### <span v-pre>`{{bgImageUrl}}`</span>
-
-Inserts a URL to the background image configured for your application. You can change this image by using the **Sign-in Configuration** option that is accessed by selecting **Settings**, and then **Appearance** from the Admin Console, but this changes the background image in all instances where the macro is used, including your custom sign-in page.
+> **Note:** There is no draft mode in the code editor. When you click **Publish**, your changes go live. If you want to test changes without impacting production, use a test or development org.
 
-If you want to just change the background image for your custom error pages, include the URL to the image instead of the macro:
+1. In the Admin Console, go to **Customizations** > **Branding**.
+2. In the **Okta-hosted Error Pages box**, click **Edit**.
+3. To open the code editor, turn on the toggle next to **Code editor**.
+4. Make changes directly in the editor. If you enter `{`, `(`, or `.` you see a list of available variables that you can use. See [Use variables](#use-variables).
+   * Click **Preview** to see your changes in a new browser window before you publish.
+   * Select **Compare with published version** to see the difference between your edited version and the published version. You can choose between a split view and a unified view.
 
-Example:
+   > **Note:** The Admin Console auto-saves your changes every few seconds. You can navigate away from the code editor and your updates remain. To discard your changes without publishing them, click **Revert changes** or turn off the toggle next to **Code editor**. The console restores the default HTML/CSS and JavaScript code.
 
-```html
-<div class="login-bg-image" style="background-image: url('https://example.com//YourBackgroundImage.png')"></div>
-```
+5. Click **Publish** to commit your changes.
 
-### <span v-pre>`{{orgLogo}}`</span>
+## Use variables
 
-Inserts the logo image that has been configured for your application. You can change this logo by using the **Organization Logo** option that is accessed by selecting **Settings**, and then **Appearance** from the Admin Console, but this changes the org logo in all instances where the macro is used, including your custom sign-in page.
+The following variables contain the configuration parameters for certain page elements. These variables inject specific content or functionality automatically.
```

Comment:
I understand why you'd suggest this, but the error pages use Mustache instead of VTL. It's an older Java-based protocol. I do link to it in the note that precedes the table with the variables, but maybe I should add it here, too? I'll use the same format as the text you provided.

Reply:
@brentschaus-okta I think it's worth pointing out this is Mustache rather than VTL explicitly. They look very similar.

Reply:
Agreed, and done.